### PR TITLE
style: lowercase graph titles (and other content changes)

### LINKED ISSUE #36: style: lower case all graph titles

### PR DESCRIPTION
**Pull request by jaxonL**
### changes
- lowercase graph titles
  - unless the word is a proper noun or acronym
  - graph labels keep their capitalisations
- content changes:
  - move cat/dog person description to misc section
  - use US instead of USA ([see here](https://english.stackexchange.com/questions/159018/the-usa-vs-the-us)]
  - some rewording of graph titles and related text
- replace male/female with men/women
- add punctuation to big spans of text (not the ones meant as a title, as in the stories section)

### other
- linechart label improvements! will now increase the svg size by that little bit that we need for it to display our labels instead of cutting it off (happened when i was testing while zoomed)

  before:
  image

  after:
  image

- fix some offsets in the data (men/women labeling in salary, co-op types typo)
closes #36 .

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,23 +1,27 @@
 # Class Profile Web
+
 =========================
 
 ## About
-The Software Engineering Class Profile is a student-initiative focused on showcasing what the program was like for students. It covers content covering co-op, background, academics, lifestyle, future plans, and answers questions such as “Does it matter if I started coding later than everyone else?” or “How much debt do students graduate with?”.
 
-You can view the class profile at [https://classprofile.andyzhang.net](https://classprofile.andyzhang.net).
+The Software Engineering Class Profile is a student-initiative focused on showcasing what the program was like for students. It covers content covering co-op, background, academics, lifestyle, future plans, and answers questions such as "Does it matter if I started coding later than everyone else?" or "How much debt do students graduate with?".
```
Comment (roxelo) [inline README.md]: We should remove `""Does it matter if I started coding later than everyone else?"` as we actually don't have any graph answering this question anymore

File: content/background.html [hunk 2]
```diff
@@ -87,7 +87,7 @@ <h3> International Students </h3>
   International students came from countries such as China and India.
 </p>
 <div class="half second">
-  <span class="graph-title">Proportion of international students</span>
+  <span class="graph-title">nationality breakdown</span>
```
Comment (roxelo) [inline content/background.html]: I don't think we should use the word nationality here since the graph shows only the number of international students and domestic students. And students could have multiple nationalities

File: content/background.html [hunk 3]
```diff
@@ -240,10 +240,10 @@ <h3>CEGEP</h3>
   <span class="big">
   All students from Quebec attended CEGEP partially, meaning they did not graduate.
   </span>
-  Among respondents, no students from outisde of Quebec attended CEGEP.
+  Among respondents, no students from outside of Quebec attended CEGEP.
 </p>
 <div class="half second">
-  <span class="graph-title">Quebec Students & CEGEP</span>
+  <span class="graph-title">quebec students & CEGEP</span>
```
Comment (roxelo) [inline content/background.html]: I think Quebec needs to be uppercased

**Review by roxelo — COMMENTED** [review]
Small nits, also I am wondering if we should also change the graph titles to use the word `respondent` instead of `student` now since we are changing all the graph title already?

File: content/background.html (hunk 2 above)
Comment (jaxonL) [reply content/background.html]: changed to "domestic (Canadian) vs international student breakdown" in 30296c1. the reason i had changed it was because the graph displays "Canadian" and "international", which imo didn't quite match the graph title..? you're right tho, in that we only see whether a student was canadian or not (open to other suggestions of graph titles)

**Comment by jaxonL** [comment]
> 
> 
> Small nits, also I am wondering if we should also change the graph titles to use the word `respondent` instead of `student` now since we are changing all the graph title already?

i do think we use this interchangeably; maybe we specify this at the beginning (in the introduction?) so that people don't get the wrong idea? i do feel respondents to come off as pretty "cold" when reading graph titles

File: README.md (hunk 1 above)
Comment (jaxonL) [reply README.md]: switched it for another question :>

File: content/coop.html [hunk 4]
```diff
@@ -311,23 +312,22 @@ <h3> Gender </h3>
   Women are paid less than men on average for co-op terms.
   </span>
 
-  Although Women are paid less than men, this is not necessarily related to a gender bias. Some
-  possible reasons could be different interests in terms of work location, type of work, etc among
+  Although women are paid less than men, this is not necessarily related to a gender bias. Some
+  possible reasons could be due to different interests in terms of work location, type of work, etc. among
```
Comment (AkshayPall) [inline content/coop.html]: I would rather reword this to say "it is difficult to determine what portion of the compensation difference is due to gender vias vs. student interest etc."

Currently, it sounds too dismissive of gender-based income discrimination, even though I know we make this clear in the following paragraph.

File: content/coop.html [hunk 5]
```diff
@@ -311,23 +312,22 @@ <h3> Gender </h3>
...
   men and women respondents in the class.
 
-  NOTE: there were about 42-30 men and 9-8 women who responded to coop salaries
+  NOTE: there were about 42-30 men and 9-8 women who responded to coop salaries.
```
Comment (AkshayPall) [inline content/coop.html]: why is this 42-30 and 9-8 instead of 30-42 and 8-9?

**Review by AkshayPall — CHANGES_REQUESTED** [review]
Looks great! I just have the 2 comments above then let's merge!

Thank you for making it so squeaky clean 🧼✨

File: content/coop.html (hunk 5 above)
Comment (jaxonL) [reply content/coop.html]: i'll take a look; this was the original wording, so i hadn't questioned it haha (i had assumed it was something related to the terms, but i may be wrong)

Comment (roxelo) [reply content/coop.html]: Since the graph provides percent I just wanted to clarify that the responses for each term and gender varied between 42 to 30 men and 9 to 8 women, I didn't think of the order

**Review by roxelo — APPROVED** [review]
